validator.rb

```diff
@@ -0,0 +1,27 @@
+require 'pry'
+
+class StringValidator
+
+  BRACKET_HASH= {
+    "(" => ")",
+    "{" => "}",
+    "[" => "]"
+  }
+
+  def validate(string)
+    return false if string.length < 2
+    brackets = []
```

Comment:
Do you think that `each_char` would be more or less efficient then chars-ing and then mapping?